src/Symfony/Component/Config/Resource/ReflectionClassResource.php

```diff
@@ -182,7 +190,7 @@ private function generateSignature(\ReflectionClass $class): iterable
             yield print_r($defaults, true);
         }
 
-        if ($class->isAbstract() || $class->isInterface() || $class->isTrait()) {
+        if ($class->isAbstract() || $class->isInterface() || $class->isTrait() || $class->isEnum()) {
```

Comment:
what about removing this check? I don't see why we should have this

Reply:
Removed.